micropub.php

```diff
@@ -810,31 +810,44 @@ private static function check_error( $result ) {
 			return $result;
 		}
 
+
+		public static function get_micropub_endpoint() {
+			return apply_filters( 'micropub_endpoint', site_url( '?micropub=endpoint' ) );
```

Comment:
I will do that. Having the three different versions come from a function as opposed to hard-coded ensured that they'd always be set identically. And as long as there was a function required, the filter made sense.